mapie/tests/test_time_series_regression.py

```diff
@@ -518,3 +518,21 @@ def test_method_error_in_update(monkeypatch: Any, method: str) -> None:
     with pytest.raises(ValueError, match=r".*Invalid method.*"):
         mapie_ts_reg.fit(X_toy, y_toy)
         mapie_ts_reg.update(X_toy, y_toy)
+
+
+def test_aci_method_in_fit() -> None:
```

Comment:
You should probably see if you can make the test more generic... for example, `"enbpi"` should also not have its method changed right?